.github/workflows/avm.res.insights.action-group.yml

```diff
@@ -71,7 +71,7 @@ jobs:
   #   Call reusable workflow   #
   ##############################
   call-workflow-passing-data:
-    name: "Module"
+    name: "Run"
```

Comment:
Shouldn't we have this change in all caller workflows?

Reply:
Yes

Reply:
done